shared/ui/core/src/main/java/com/ivy/ui/FormatMoneyUseCase.kt

```diff
@@ -19,11 +22,24 @@ class FormatMoneyUseCase @Inject constructor(
     private val withDecimalFormatter = DecimalFormat("###,###.00", DecimalFormatSymbols(locale))
 
     suspend fun format(value: Double): String {
-        val showDecimalPoint = features.showDecimalNumber.isEnabled(context)
+        when (value >= MILLION) {
```

Comment:
Make the function accept `shortenAmount` parameter, shortening isn't always desired

Reply:
Done